.github/workflows/pull_request.yml

```diff
@@ -96,8 +115,23 @@ jobs:
       # see https://stackoverflow.com/a/60984318/2886726
       run: |
         git ls-files taxonomies/ | xargs -I{} git log -1 --date=format:%Y%m%d%H%M.%S --format='touch -t %ad "{}"' "{}" | bash
+    - uses: actions/cache/restore@v4
+      id: cache
+      with:
+        path: ./build-cache
+        key: taxonomies-${{ hashFiles('taxonomies/**') }}
+        restore-keys: taxonomies-
+    - if: steps.cache.outputs.cache-hit == 'true'
+      name: Prepare build_cache volumes
+      run: |
+        docker volume create \
+                   --label com.docker.compose.project=po_off \
+                   --label com.docker.compose.version=2.29.7 \
+                   --label com.docker.compose.volume=build_cache \
+                   po_off_build_cache
+        docker container run --rm -v ./build-cache:/from -v po_off_build_cache:/to alpine ash -c 'cd /from ; cp -av . /to'
     - name: Download backend image from artifacts
-      uses: ishworkh/docker-image-artifact-download@v1
+      uses: ishworkh/container-image-artifact-download@v2.0.0
```

Comment:
Is this needed for this particular change or is it just a general version update?